task1/index.js

```diff
@@ -0,0 +1,108 @@
+//А
+
+const myMap = (collection, callback) => {
+  const result = [];
+  for (let item of collection) {
+    result.push(callback(item));
+  }
+  return result;
+};
+
+const myFilter = (collection, callback) => {
+  const result = [];
+  for (let item of collection) {
+    if (callback(item)) {
+      result.push(item);
+    }
+  }
+  return result;
+};
+
+const myReduce = (collection, callback, init) => {
+  let acc = init;
+  for (let item of collection) {
+    acc = callback(acc, item);
+  }
+  return acc;
+};
+
+//Б
+
+const notes = [
```

Comment:
Let's rename this constant using uppercase. When we declaring a variable that contains a collection of items or plain value (string, number), without any getters/setters/calculations -> uppercase
constant with getters/setters/calculations -> camelcase
```suggestion
const NOTES = [
```